src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js

```diff
@@ -23,6 +54,9 @@ const ApplicationPreview = ({ header, body, privateInfo, onClick }) => (
         <Link to="/review">Go back and edit</Link>
       </div>
     </div>}
+    <div>
+      <Changes {...body}/>
```

Comment:
it renders conditionally on unassessed case studies which public profile shouldn't have but I'll take a look at that